packages/@vuepress/core/lib/node/build/index.js

```diff
@@ -158,7 +158,7 @@ module.exports = class Build extends EventEmitter {
       console.error(logger.error(chalk.red(`Error rendering ${pagePath}:`), false))
       throw e
     }
-    const filename = decodeURIComponent(pagePath.replace(/\/$/, '/index.html').replace(/^\//, ''))
```

Comment:
What's the intention to do this change?

Reply:
It will change the output of the console.